openy_gated_content.module

```diff
@@ -193,6 +193,10 @@ function openy_gated_content_form_alter(&$form, FormStateInterface $form_state,
     'eventinstance_live_stream_edit_form',
     'eventinstance_virtual_meeting_edit_form',
   ];
+  $events_bundle = [
+    'live_stream',
+    'virtual_meeting',
+  ];
```

Comment:
I don't believe we need this hard-coded array here. We have config to set entity types and bundles - https://github.com/fivejars/openy_gated_content/blob/1.4/config/install/openy_gated_content.settings.yml#L35